tests/store/test_file_filters.py

```diff
@@ -0,0 +1,27 @@
+
+from typing import List
+from housekeeper.store import Store
+from housekeeper.store.models import File
+from housekeeper.store.filters.file_filters import filter_files_by_id
+
+
+def test_filter_files_by_id_returns_the_correct_file(populated_store: Store):
+    """Test getting collaboration by internal_id."""
+
+    # GIVEN a store with a file
+    file: File = populated_store.session.query(File).first()
```

Comment:
I asserted that the Return from the query is a query and then later asserted on a .first() just to keep in check that query functions should return queries.

What is the consensus here? @seallard @Vince-janv @diitaz93 @henrikstranneheim 

```Python
file: Query = populated_store.session.query(File)
assert isInstance(file, Query)
```